pkgs/applications/window-managers/sway/wrapper.nix

```diff
@@ -17,6 +18,7 @@ with lib;
 
 let
   sway = sway-unwrapped.override { inherit isNixOS enableXWayland; };
+  swayBinary = if useSetcapWrapper then "sway-setcap" else "${sway}/bin/sway";
```

Comment:
Not sure I like getting the binary from PATH. What do other packages do here?

Reply:
I don't really like that either but IIRC most other packages I found do it like this.

We could also use `/run/wrappers/bin/sway-setcap` but we should rather use `config.security.wrapperDir` via the module system (e.g., as `wrapper.nix` parameter: `setcapWrapper = "${config.security.wrapperDir}/sway-setcap"` (default: `null`) - instead of using `useSetcapWrapper`). I think that's probably better.